File: Easy/412-Fizz-Buzz/Solutions.cpp
class Solution
{
public:
    vector<string> fizzBuzz(int n)
    {
        vector<string> hasil;
        for (int i = 1; i <= n; i++)
        {
            if (i % 3 == 0 && i % 5 == 0)
                hasil.push_back("FizzBuzz");
            else if (i % 3 == 0)
                hasil.push_back("Fizz");
            else if (i % 5 == 0)
                hasil.push_back("Buzz");
            else
                hasil.push_back(to_string(i));
        }
        return hasil;
    }
};

File: Easy/412-Fizz-Buzz/Solutions.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        hasil = []
        for i in range(1,n+1):
            if i % 3 == 0 and i % 5 == 0: hasil.append("FizzBuzz")
            elif i % 3 == 0: hasil.append("Fizz")
            elif i % 5 == 0: hasil.append("Buzz")
            else: hasil.append(str(i))
        return hasil